coder_sniffer/Drupal/Test/bad/BadUnitTest.php

```diff
@@ -334,7 +334,7 @@ public function getErrorList($testFile)
                         714 => 1,
                         716 => 1,
                         717 => 1,
-                        721 => 1,
+                        722 => 1,
```

Comment:
we should leave the BadUnitTest as is - it is there for historical reasons and we should not modify it if we don't have to.

Reply:
I reverted the changes.